treeherder/seta/job_priorities.py

```diff
@@ -87,7 +55,7 @@ def _gecko_decision_task_request(project):
             if task_request.counter % jp.priority != 0:
                 job_priorities.append(jp)
 
-    return _process(build_system='taskcluster', job_priorities=job_priorities)
+    return _process(project, build_system='taskcluster', job_priorities=job_priorities)
```

Comment:
I mean this change was to fix the repo always being mozilla-inbound regardless of what was specified. However no tests failed, even with this missing, so we don't have full test coverage.